install/kubernetes/helm/istio/charts/ingress/templates/deployment.yaml

```diff
@@ -32,8 +32,10 @@ spec:
           - ingress
           - -v
           - "2"
-          - --discoveryAddress
+          - --discoveryPlainAddress
           - {{ .Release.Name }}-pilot:15003
+          - --discoveryMtlsAddress
```

Comment:
I'm not sure I get this: why would we need both ? Based on policy we should select only one.